docs/02.API-REFERENCE.md

```diff
@@ -3287,6 +3309,100 @@ bool foreach_function (const jerry_value_t prop_name,
 - [jerry_object_property_foreach_t](#jerry_object_property_foreach_t)
 
 
+# Input validatator functions
+
+## jerry_is_utf8_string_valid
+
+**Summary**
+
+Validate UTF-8 string.
+
+**Prototype**
+
+```c
+bool
+jerry_is_utf8_string_valid (const jerry_char_t *utf8_buf_p, /**< UTF-8 string */
```

Comment:
Good to me. Should we rename the internal functions too? `lit_is_valid_utf8_string`?